sooyoung/models/userDao.js

```diff
@@ -19,6 +19,41 @@ const createUser = async (username, email, profileImage, password) => {
   }
 };
 
+const getHashedPassword = async (email) => {
+  try {
+    const [hashedPassword] = await appDataSource.query(
+      `SELECT password
+        FROM users
+        WHERE email = ?;
+      `,
+      [email]
+    );
+    return hashedPassword.password;
+  } catch (err) {
+    const error = new Error('INVALID_EMAIL_INPUT');
+    error.statusCode = 500;
+    throw error;
+  }
```

Comment:
```suggestion
  } catch (err) {
    console.log(err)
    
    const error = new Error('INVALID_EMAIL_INPUT');
    error.statusCode = 500;
    throw error;
  }
```
client에서는 서버의 에러 메세지가 그대로 보이지 않게 지금처럼 처리를 해주셔야합니다. 👍
다만 개발자가 디버깅 할 수 있도록, catch block에서 받은 err 인자를 console.log로 한 번 표시 한뒤 새로운 에러 객체를 만들어 throw하도록 예시를 참고해 console.log 코드 추가해주세요.